app/presenters/publishing_api/world_location_news_page_presenter.rb

```diff
@@ -17,10 +17,10 @@ def content
       content.merge!(
         description: description,
         details: {},
-        document_type: "placeholder_world_location_news_page",
+        document_type: "placeholder_world_location_news_page", # TODO: do we also want to change this document type? What are the implications
```

Comment:
I'll remove this comment - decided to split out updating the document type into a different ticket, as it requires a bit more work than just changing this string.

Reply:
👍